### PR TITLE
Rs 2022 01 25

### PR DESCRIPTION
**Pull request by rschlaikjer**
good lord this file was long

### CODE REVIEW

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 1]
```diff
@@ -162,22 +181,22 @@ Even from seeing her back alone, I can clearly grasp the extent of the hostile a
...
 -- Page 286, Offset 17272.
 -- 彼女が何をしようとしているのか、言葉にするまでもなく判りきっている。
-There was no need for her to put it into words for me to understand what she was about to do.
+I know what she's about to do even without hearing it put into words.
 }
 [sha:63da4b06040e0c7748b8cf8894a9994fea4fcdde]{
 -- Page 286, Offset 17273.
 -- アルクェイドから放たれているのは一点の濁りもない『殺意』だった。
-The thing emanating from Arcueid was, without a shadow of a doubt, her 'killing intent'.
+What emanates from Arcueid right now is an unmistakable 'killing intent'.
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:194]: Not so sure about 'killing intent'. Could be 'intent to kill', but I think I might just favor 'murderous intent'.

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 2]
```diff
@@ -802,17 +885,18 @@ After taking notice of my serious anger, I realized something.
...
 -- Page 331, Offset 17399.
 -- やっぱり俺は、このどうしようもなくやっかいな女を、放っておけない。
-There's no way that I can let this troublesome woman go around and do as she pleases.
+This woman truly is an unbelievable pain in the ass, but in the end I just can't leave her alone.
 }
 [sha:51c59285d6ffa66c62986346de45129659361ee3]{
 -- Page 332, Offset 17400.
 -- 「……おまえ、ホントに何してんだよ。
-"...What the hell were you thinking?
+"...What the hell is all this about?
```
Comment (Avuxo) [inline script/Arcueid/Day 7/07_00_ARC07_3E.txt:893]: revert

**Review by Avuxo — APPROVED** [review]

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 3]
```diff
@@ -1,32 +1,43 @@
 [sha:1243f5e061aee1adbf432f8c5d5294628588f5d5]{
 -- Page 10012, Offset 17239.
 -- アルクェイドは夜の街を歩いていく。
-Arcueid walks through the night city.
+// Sideways TL of 夜の街 but "night city" just doesn't sit right, am I crazy?
+// 'Nighttime city' _maybe_ but it just doesn't seem fluent
+Arcueid strides through the twilit city.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3E.txt:6]: man I really need to figure out how to get a better dictionary loaded into gjiten, or I guess use weblio more

[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3E.txt:6]: What about translating it more as the vibe than the physical location?
> Arcueid cuts through the city nightlife.

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt [hunk 4]
```diff
@@ -1,32 +1,43 @@
...
 }
 [sha:c2ab1f793ef1fb4acf6412e7a976846f372e2ff9]{
 -- Page 275, Offset 17244.
 -- 間違いない。あれは狩人の在り方だ。
-There's no doubt about it. That must've been my hunter's instincts coming into play .
+// Time to play the "what's the subject of the sentence" game
+// 狩人の在り方 reads to me as "the way a hunter ought to be". I feel like
+// there's 2 possible applications here
+// あれ is his own personal state. The disappearance of his feelings is a return to hunting form.
+// あれ is Arc. Her form is the perfect shape for a huntress.
```
[the remark this answers is not in the document]
Comment (rschlaikjer) [reply script/Arcueid/Day 7/07_00_ARC07_3E.txt:36]: Petrikow seems to agree that あれ here is indeed Arc's figure, so going to leave as-is

File: script/Arcueid/Day 7/07_00_ARC07_3E.txt (hunk 3 above)
[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 7/07_00_ARC07_3E.txt]: I think that's a good solution.

**Pull request merged**